### PR TITLE
Update README.md with release link for CLI tool

### PR DESCRIPTION
**Pull request by esthersweon**
### Public-Facing Changes

- N/A

### Description

- Update README.md with release link for CLI tool

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +31,8 @@ which is used to store the test logs under `tests/conformance/data`.
 
 A CLI tool for interacting with the format is available [here](./go/cli/mcap).
 
+Download the [latest release](https://github.com/foxglove/mcap/releases/latest).
```
Comment (wkalt) [inline README.md:34]: I wonder if maybe we should just change the link above this one to point here, and remove the new line. Probably people who are following the "is available" link are looking for a download, not to browse code.

**Review by wkalt — APPROVED** [review]

**Review by jtbandes — COMMENTED** [review]
This does not do what was intended – it links to whichever "release" has been made most recently which may not be a release of the CLI tool – for example, if someone had clicked this link on Nov 1 it would have gone to https://github.com/foxglove/mcap/releases/tag/releases%2Frust%2Fv0.9.0.

**Comment by jtbandes** [comment]
See also: https://github.com/foxglove/mcap/issues/944 & https://github.com/orgs/foxglove/discussions/796

**Comment by jtbandes** [comment]
Filed FG-6144 to fix

**Pull request merged**